src/main/java/uk/gov/hmcts/reform/coh/controller/state/StateResponse.java

```diff
@@ -36,4 +41,7 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public String getStateDesc() {
+        return WordUtils.capitalize(name.replaceAll("_", " "));
```

Comment:
Because description depends upon name, maybe this could be ran only when name changes?

```java
    public void setName(String name) {
        this.name = name;
        stateDesc = WordUtils.capitalize(name.replaceAll("_", " "));
    }

    public String getStateDesc() {
        return stateDesc;
    }
```
and in the constructor:
```java
    public StateResponse(String name, String datetime) {
        setName(name);
        this.datetime = datetime;
    }
```